test/app/database/models/numeric-id.js

```diff
@@ -0,0 +1,10 @@
+import mongoose from "mongoose";
+
+const schema = mongoose.Schema({
+  _id: {
+    type: Number,
+    required: true
```

Comment:
Sounds good to me!

If we do that, though, I think we should make sure the serialization's working before/as part of merging in support for Number ids. So, do you want to add that serialization stuff to this PR? Or keep this PR just to strings and ObjectIds, and then make a second that deals with Numbers (validation/queries + serialization)? I don't mind either way, but I could understand if you're feeling anxious to get this merged and so would prefer the former.